data/field_type_test.go

```diff
@@ -0,0 +1,45 @@
+package data_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// The slice data in the Field is a not exported, so methods on the Field are used to to manipulate its data.
+type simpleFieldInfo struct {
+	Name      string
+	FieldType data.FieldType
+}
+
+func TestFieldTypeConversion(t *testing.T) {
+	f := data.FieldTypeBool
+	s := f.ItemTypeString()
+	assert.Equal(t, "bool", s)
+	c, ok := data.FieldTypeFromItemTypeString(s)
+	require.True(t, ok, "must parse ok")
+	assert.Equal(t, f, c)
```

Comment:
there are both `require` and `assert` in a test - is this for a reason? Personally I always use `require` to stop test on error.

Reply:
this has been around for a while... i'll move them all the require